### PR TITLE
feat(raw-block-volumes): add support for raw block volumes

### PR DESCRIPTION
**Pull request by payes**
Signed-off-by: Payes Anand <payes.anand@mayadata.io>

### CODE REVIEW

File: pkg/driver/controller.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func (cs *controller) CreateVolume(
 	logrus.Infof("CreateVolume: volume: {%v} is created", req.GetName())
 	return &csi.CreateVolumeResponse{
 		Volume: &csi.Volume{
-			VolumeId:      req.GetName(),
+			VolumeId:      utils.StripName(req.GetName()),
```
Comment (prateekpandey14) [inline pkg/driver/controller.go:95]: why this change ?

Comment (payes) [reply pkg/driver/controller.go:95]: We are creating jivaVolume CR using the stripped name hence we need to send the volume ID as stripped for future references.

File: tests/yamls.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
       - name: ubuntu
-        image: prateek14/ubuntu:18.04
+        image: quay.io/payesanand/ubuntu:18.04
```
Comment (shubham14bajpai) [inline tests/yamls.go:42]: can we use the  official ubuntu image here?

File: tests/yamls.go [hunk 3]
```diff
@@ -86,6 +86,56 @@ spec:
...
+    type: Recreate
+    rollingUpdate: null
+  template:
+    metadata:
+      labels:
+        name: ubuntu-block
+    spec:
+      containers:
+      - name: ubuntu-block
+        image: quay.io/payesanand/ubuntu:18.04
```
Comment (shubham14bajpai) [inline tests/yamls.go:112]: same commnet

File: tests/yamls.go (hunk 2 above)
Comment (payes) [reply tests/yamls.go:42]: pause binaries are not present in official images

**Review by shubham14bajpai — APPROVED** [review]

**Review by prateekpandey14 — APPROVED** [review]
Lgtm
